addons/packages/pinniped/0.4.4/bundle/config/overlay/update_overlay.yaml

```diff
@@ -0,0 +1,31 @@
+#@ load("@ytt:overlay", "overlay")
+#@ load("@ytt:data", "data")
+
+#@overlay/match by=overlay.subset({"kind":"DaemonSet"}), expects="0+"
```

Comment:
pinniped doesn't use `DaemonSet`'s, so this is dead code. i understand the need to make the `update_overlay.yaml` files uniform, but should we remove this part of the overlay for packages that don't have `DaemonSet`'s?

Reply:
And if they MUST be duplicated yet consistent, I think it warrants a good comment explaining why. But I definitely prefer a solution that doesn't require the [same file duplicated across all packages](https://github.com/vmware-tanzu/community-edition/pull/2277/files#r732801550).